fixtures/fixture.until.spec.yaml

```diff
@@ -0,0 +1,30 @@
+apiCalls:
+    swagger:
```

Comment:
Can we add one test to ```apiCall.spec.js``` to make sure it does not break in the future? as off we have done with the repeat tag??? Checkout https://github.com/testbetter/openapitest/blob/master/specs/apiCall.spec.js#L69 for instance

Reply:
@tasfin-hasan Can we add the unit tests to condition?????

Reply:
done